tests/schemas/valid/test_section_enabled.json

```diff
@@ -27,25 +22,27 @@
     }
   ],
   "questionnaire_flow": {
-    "type": "Hub",
+    "type": "Linear",
     "options": {
-      "required_completed_sections": ["default-section"]
+      "summary": {
+        "collapsible": false
+      }
     }
```

Comment:
Should use `Hub` as although not enforced yet, multiple section should be hub.

Reply:
Changed to Hub